File: engine/model/src/main/scala/tools/packet/pq/ariant/UpdateAriantRanking.scala
package tools.packet.pq.ariant

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class UpdateAriantRanking(private var _scores: java.util.List[AriantScore]) extends PacketInput {
  def scores: java.util.List[AriantScore] = _scores

  override def opcode(): SendOpcode = SendOpcode.ARIANT_ARENA_USER_SCORE
}

File: engine/engine-model/src/main/java/tools/packet/pq/ariant/UpdateAriantRanking.java
package tools.packet.pq.ariant;

import java.util.List;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record UpdateAriantRanking(List<AriantScore> scores) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.ARIANT_ARENA_USER_SCORE;
   }
}